Yoast/wordpress-seo
# Fix incorrect output in translation `.php` translator comments

# Note: This should be implemented in the first release - patch or next regular release.

_This PR has been based on `master`_

## Summary

The tool that collects the translator comments from JavaScript files cannot parse them in the best way.
These still contain `*/` elements at the end of the sentences.

There is another problem regarding one specific translation, where the translator comment line is being duplicated - resulting in two `*/` occurrences in the same line.

The previous functionality only replaces 1 occurrence in the line, thus leaving the last one.
This causes linting problems when applied to this file.

This PR can be summarized in the following changelog entry:

* [not-user-facing] Makes sure all the `*/` entries are stripped from translator comments

## Relevant technical choices:

* Use regular expression to remove all `*/` occurrences from the translator comments

## Test instructions

This PR can be tested by following these steps:

* Run `grunt build:i18n:potFiles`
* Inspect the `languages/yoast-seo-js.php` file and see no `*/ */` occurrences are present anymore

## UI changes

## Quality assurance


Fixes #12279
